routes/yt_commenter.py

```
@@ -0,0 +1,113 @@
+
+    Looks up the transcript in the database by video URL, retrieves the
+    summary, and posts it as a top-level YouTube comment.
+    """
+    from app.services.yt_commenter import YouTubeCommenterService
+
+    try:
+        service = YouTubeCommenterService()
+        result = service.comment_on_video(request.video_url)
+        return {"status": "success", "data": result}
+    except ValueError as e:
+        raise HTTPException(status_code=404, detail=str(e))
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)